kong/plugins/zipkin/schema.lua

```diff
@@ -3,10 +3,10 @@ local typedefs = require "kong.db.schema.typedefs"
 return {
 	name = "zipkin",
 	fields = {
+		{ run_on = typedefs.run_on { default = "all" } },
```

Comment:
The original code included `one_of = { "all" }`, to force that only "all" would be valid for this plugin. Was this removed deliberately?

Reply:
Yes. There's no reason to disallow someone logging to zipkin from only one side.